src/main/java/AmountOfTimeBeforeOrAfterCourseCompletion.java

```diff
@@ -92,29 +92,16 @@
         return daysAndHoursBeforeOrAfterCourseCompletion;
     }
 
-    public static DurationBeforeOrAfterCourseCompletion calculateAmountOfTimeBeforeOrAfterCourseCompletion(LocalDateTime endDate, LocalDateTime currentDate, int coursesDuration) {
-        long nightTime = Duration.between(endDate.withHour(endWorkingHour), currentDate.withHour(startWorkingHour)).toHoursPart();
-        long amountOfWholeDaysBetweenEndAndCurrent = Duration.between(currentDate, endDate).toDays();
-        long amountOfWholeHoursBetweenEndAndCurrent;
-        int remainingHoursOfEducation = coursesDuration % DetermineCourseCompletion.workHoursPerDay;
-
-        LocalDate endDateLocalDate = endDate.toLocalDate();
-        LocalDate currentDateLocalDate = currentDate.toLocalDate();
-
-        if (endDateLocalDate.plusDays(Math.abs(amountOfWholeDaysBetweenEndAndCurrent)).compareTo(currentDateLocalDate) < 0 && remainingHoursOfEducation != 0) {
-            amountOfWholeHoursBetweenEndAndCurrent = Duration.between(currentDate, endDate.withHour((remainingHoursOfEducation + startWorkingHour))).toHoursPart() + nightTime;
-        } else if (endDate.isAfter(currentDate) && remainingHoursOfEducation == 0) {
-            amountOfWholeHoursBetweenEndAndCurrent = Duration.between(currentDate, endDate).toHoursPart();
-        } else {
-            amountOfWholeHoursBetweenEndAndCurrent = Duration.between(currentDate, endDate.withHour((remainingHoursOfEducation + startWorkingHour))).toHoursPart();
-        }
-
-        DurationBeforeOrAfterCourseCompletion durationBeforeOrAfterCourseCompletion = new DurationBeforeOrAfterCourseCompletion((int) amountOfWholeDaysBetweenEndAndCurrent, (int) amountOfWholeHoursBetweenEndAndCurrent);
-
-        if (amountOfWholeDaysBetweenEndAndCurrent >= 0 && amountOfWholeHoursBetweenEndAndCurrent > 0) {
-            System.out.println("Training is not finished. " + amountOfWholeDaysBetweenEndAndCurrent + " d " + amountOfWholeHoursBetweenEndAndCurrent + " hours are left until the end.");
+    public static DurationBeforeOrAfterCourseCompletion calculateAmountOfTimeBeforeOrAfterCourseCompletion(LocalDateTime endDate, LocalDateTime currentDate) {
+        int daysBetween = (int) Duration.between(currentDate, endDate).toDays();
+        int hoursBetween = Duration.between(currentDate, endDate).toHoursPart();
+        DurationBeforeOrAfterCourseCompletion durationBeforeOrAfterCourseCompletion = new DurationBeforeOrAfterCourseCompletion(daysBetween, hoursBetween);
+        if ((daysBetween > 0 && hoursBetween > 0) || (daysBetween == 0 && hoursBetween > 0) || (daysBetween > 0 && hoursBetween == 0)) {
```

Comment:
let's not print anything inside this method. let's do printing inside `main` method only.

Reply:
how main method will determine if course completed or not?

Reply:
the idea is that the `calculate` method should do the calculation only